api.jsx

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const ncNewsAPI = axios.create({
+  baseURL: `https://northcoders-news-ykvv.onrender.com/api`,
+});
+
+export const fetchArticles = async (topic) => {
+  try {
+    const response = await ncNewsAPI.get(`/articles?topic=${topic}`);
+    console.log("POST request successful, response:", response);
```

Comment:
Ensure you're removing console logs before pushing your code 👍

Reply:
Will do sorry - one day, that instruction will stick!!